packages/b/binutils/xmake.lua

```diff
@@ -19,6 +19,27 @@ package("binutils")
         add_extsources("brew::binutils")
     end
 
+    on_fetch("@linux", "@macosx", "@msys", function (package, opt)
+        if opt.system then
+            if package:is_binary() then
+                if package:find_tool("ld") then
```

Comment:
We should check a few more tools, just checking ld doesn't guarantee that the binutils are already there, and we should exclude the windows platform.

Reply:
> We should check a few more tools, just checking ld doesn't guarantee that the binutils are already there  

Ok I'll do that  

> and we should exclude the windows platform  

Do you mean mysys ? I thought it would work because it's based on Arch, plus their website references a [binutils package](https://packages.msys2.org/package/binutils).